pkg/api/v1/types.go

```diff
@@ -1241,6 +1243,10 @@ type PodSpec struct {
 	ImagePullSecrets []LocalObjectReference `json:"imagePullSecrets,omitempty" patchStrategy:"merge" patchMergeKey:"name"`
 }
 
+// PodSecurityContext holds pod-level security attributes and common container settings.
+type PodSecurityContext struct {
```

Comment:
@thockin It means I'm agreeing, but I'm building on top of this PR.  I'm fine with squashing into a single commit once `PodSecurityContext` has at least one field in the v1 API, but I really want to make some progress on what we're going to build on top of the commit in this PR.